File: 백준/Bronze/1934. 최소공배수/최소공배수.py
#주석처럼 하지 말고 lcm 함수로 시간단축하기
'''t = int(input())

for i in range(t):
    a, b = map(int, input().split())
    
    lcm = min(a,b)
    
    while True:
        if lcm % a == 0 and lcm % b == 0:
            print(lcm)
            break
        else:
            lcm += 1'''
import math

t = int(input())

for i in range(t):
    a, b = map(int, input().split())
    print(math.lcm(a, b))

File: 백준/Bronze/1934. 최소공배수/최소공배수.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int T = Integer.parseInt(br.readLine());

        for (int i = 0; i < T; i++) {
            st = new StringTokenizer(br.readLine());

            int A = Integer.parseInt(st.nextToken());
            int B = Integer.parseInt(st.nextToken());

            int gcd = GCD(Math.max(A, B), Math.min(A, B));

            int result = A * B / gcd;
            System.out.println(result);
        }
    }

    static int GCD(int A, int B) {
        if (A % B == 0) {
            return B;
        }

        return GCD(B, A % B);
    }
}